client/memez/projects/memez/src/app/stores/views/layout.store.ts

```diff
@@ -1,16 +1,34 @@
-import {Injectable} from "@angular/core";
-import {RootStore} from "../root.store";
+import {Injectable}    from "@angular/core";
+import {RootStore}     from "../root.store";
 import {autorun, toJS} from "mobx";
+import {Router}        from "@angular/router";
 
 @Injectable({providedIn: "root"})
 export class LayoutStore {
 
 
-  constructor(public root: RootStore) {
+  constructor(
+    public root: RootStore,
+    public router: Router) {
     this.root.ls = this;
     window['layoutStore'] = this;
     autorun(() => {
-      console.log(JSON.stringify(toJS(this.root.us.users)))
+
     })
   }
+
+  async routeToFeed() {
+    await this.router.navigateByUrl('feed')
+  }
+
+  async routeToProfile() {
+    await this.router.navigateByUrl('profile')
+  }
+
+  async routeToLogin() {
+    await this.router.navigateByUrl('login')
+    window.location.reload();
```

Comment:
why window.reload?? This should be avoided whenever possible

Reply:
It's temporary just to get the user to log off... going to change it as soon as i implement proper auth methods.